pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/module/ModuleVersionUploadTest.java

```diff
@@ -105,10 +105,10 @@ void createFirstVersionUnicodeIso() throws Exception {
         ModuleVersionUpload upload = new ModuleVersionUpload(asciidoctorService);
         Map<String, Object> params = newHashMap();
         params.put("locale", Locale.SIMPLIFIED_CHINESE.toString());
-        params.put("asciidoc", "å\u008D\u0097äº¬é\u0098²ç\u0096«ç\u008E°å\u009Cº");
+        params.put("asciidoc", "d'agua per toles partes, por exemplu: Sicilia y Cuba; al otrudía  decatáronse  d'ú  veníen  les  voces:  de  dientro  la  casa;  nun  foi  aaguantar aquel perru: vendiólu.");
         slingContext.request().setParameterMap(params);
         slingContext.request().setResource(new NonExistingResource(slingContext.resourceResolver(), "/content/repositories/test_workspace/entities/new/proc_module"));
-        slingContext.request().setCharacterEncoding(StandardCharsets.ISO_8859_1.toString());
+        slingContext.request().setCharacterEncoding(StandardCharsets.UTF_8.toString());
```

Comment:
Echoing my previous concern - it's nice that we have a test to confirm that UTF-8 works, but we should also have tests to make sure that other encodings work as well. It seems like with these changes, we're forcing all uploads to use UTF-8 and I don't think that's the correct design decision for the system.

Reply:
@benradey Thanks for this comment,
This test case was added for accented characters, we can push negative test case, to have your concern resolved.

Reply:
@rednitish, rather than overwriting the ISO_8859_1 test, can you copy/paste it so that your new test for the accented characters becomes an additional test?